spyderlib/plugins/help.py

```diff
@@ -185,7 +185,7 @@ def __init__(self, parent):
 
         self.webview = FrameWebView(self)
         self.find_widget = FindReplace(self)
-        self.find_widget.set_editor(self.webview)
+        self.find_widget.set_editor(self.webview._webview)

```

Comment:
This was a change made by @goanpeca, so the real `webview` widget is saved in a private variable.

I'll a new method to access it :-)